src/Features/Core/Portable/FileHeaders/FileHeader.cs

```diff
@@ -0,0 +1,92 @@
+﻿// Copyright (c) Tunnel Vision Laboratories, LLC. All Rights Reserved.
```

Comment:
is having this copyright be against Tunnel Vision ok?

Reply:
No, this is supposed to use the one from Roslyn. If only we had a way to validate these...

Reply:
done